src/concepts/terminology.md

```diff
@@ -86,7 +86,9 @@ Intuitively, how many days per week do users use the product? Among profiles act
 
 ## Metric
 
-A metric is anything that you want to (and can) measure. This differs from a dimension which is a qualitative attribute of data.
+In general: a metric is anything that you want to (and can) measure. This differs from a dimension which is a qualitative attribute of data.
+
+In the context of [Glean](#glean), a metric refers to an instrumented measure for a specific aspect of the product (similar to a [probe](#probe) in Firefox).
```

Comment:
```suggestion
In the context of [Glean](#glean), a metric refers to an instrumented measure for a specific aspect of the product (similar to a [probe](#probe) in Firefox Telemetry).
```

Glean is now also in Firefox, but we still differentiate the two. At some point in the far feature we hopefully have only one left.